### PR TITLE
Broadcast event in viewer

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

**Comment by erwinmombay** [comment]
@dvoytenko you think we should render a different doc for contrast?

File: examples/viewer.html [hunk 1]
```diff
@@ -343,11 +395,30 @@
...
+      var anchor = document.getElementById('show-container' + id + '-anchor');
+      anchor.onclick = function() {
+        showContainer(id);
+      };
+    }
+
+    function showContainer(id) {
+      log('Show container: ', id);
+      var viewerEl = document.querySelector('viewer');
+      viewerEl.setAttribute('data-show-container', id);
```
Comment (erwinmombay) [inline examples/viewer.html]: why did we have to go through the attribute route instead of a class?

**Comment by ericfs** [comment]
LGTM

**Comment by dvoytenko** [comment]
@erwinmombay would like a different document, but I only have :) Eventually we will just allow URLs to be passed as arguments to this sample.

File: examples/viewer.html (hunk 1 above)
Comment (dvoytenko) [inline examples/viewer.html]: Only for convenience. Easier to set a single value than toggle different classes on/off.

**Comment by erwinmombay** [comment]
@dvoytenko LGTM

**Pull request merged**